plugins/client/io.ktor/client-sse/3.0.0-beta-1/documentation.md

```diff
@@ -0,0 +1,13 @@
+Client "Server-sent Events" (SSE) plugin for consuming events sent using the 
```

Comment:
Client "Server-sent Events" (SSE) -> Server-Sent Events (SSE) plugin for the Ktor client.